### ISSUE TITLE
Worms Armageddon (GBC) is very glitchy. Water not rendered, tiles loading incorrectly. 

### ISSUE THREAD

**Issue by advert665** [frederic-mahe/Analogue-Pocket#103]
Worms Armageddon on gameboy colour is very glitchy on the AP. The water does not load, and scrolling around the map using the select key causes graphical glitches with tiles loading in incorrectly. Image 1 below shows the worm drowning where water should be rendered. Image 2 shows the tile glitches that occur when you scroll using select. These same issues do not occur on original hardware.

![Screenshot 2022-11-22 at 20 31 25](https://user-images.githubusercontent.com/77785968/203415275-d2fce07d-7906-476b-8bed-05463341e20c.png)
![Screenshot 2022-11-22 at 20 32 08](https://user-images.githubusercontent.com/77785968/203415333-42acc58f-898e-4aab-b1ac-c457a4ba8d9b.png)

**Comment by CommanderGrog** [comment]
I can confirm this also happens with the reproduction cart by Limited Run Games.

**Comment by andreevgy** [comment]
Can confirm it as well, happens to me with original cart. Does not happen on FPGBC though.

**Comment by arhneu** [comment]
Disappointingly, this is still an issue, two years on. Especially with the game recently re-released officially, it's a shame...

**Comment by CommanderGrog** [comment]
Yes, that's true. Especially since the game is running flawlessly with the available OpenFPGA GBC-cores. I'd really love to be able to play my real cartridge, though, since this is, what I bought the pocket for, and since lots of memories are connected to it.
